src/miniwiki.rs

```diff
@@ -284,4 +290,14 @@ fn render_arguments(parameters: &Vec<DumpFunctionParameter>) -> Fragment {
                 }
             </div>
         )))
+}
+
+fn member_element_class<'a>(tags: &'a Vec<String>, main_class: &str) -> String {

```

Comment:
Taking `&Vec<T>` is kinda funky, we should accept `&[String]` here instead. It should convert automatically just fine!

We also don't need the lifetime annotations on this function, since we aren't returning anything with a lifetime that needs to be disambiguated.

```suggestion
fn member_element_class(tags: &[String], main_class: &str) -> String {
```

Reply:
I think this is still needed with BTreeSet